### ISSUE TITLE
defaults for options.txt for new instances

### ISSUE THREAD

**Issue by alexmo1997** [PolyMC/PolyMC#613]
### Role

I play modded and vanilla Minecraft, across different versions.

### Suggestion

add a option to set defaults for options.txt options for new instances

### Benefit

I don't have to rebind keys and set gui size everytime I create a new instance

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

Just copying options.txt from other versions is firstly cumbersome, as you have to always manually go into the dirs and copy the file, and it also doesn't always work as keybinds get renamed over versions.

**Comment by txtsd** [comment]
I would like this too. I have custom keybinds for chat, inventory, and swap hands, that I don't want to manually set on every new instance.

**Comment by OhaDerErste** [comment]
would be sick!

**Comment by ryanccn** [comment]
Could symlinks work, i.e. `globaloptions.txt` in PolyMC folder linking to `options.txt`

**Comment by alexmo1997** [comment]
I don't think so, as it seems the `options.txt` format/option names in there changed across versions.

**Comment by ryanccn** [comment]
hmm but if this problem exists then global options couldn't work in any case.

maybe different global options files for each version?

**Comment by ryanccn** [comment]
how would the globaloptions.txt file be edited though... manually by the user? PolyMC new UI?

**Comment by ryanccn** [comment]
if we're doing new UI to edit options we might as well make it for all instances :D

**Comment by alexmo1997** [comment]
> if we're doing new UI to edit options we might as well make it for all instances :D

Well that's kind of what I meant.
In an ideal world, it should set the options.txt according to the versions of the instances
